qlever.py

```diff
@@ -0,0 +1,441 @@
+#!/usr/bin/python3
```

Comment:
On MacOS, I have a newer version of python installed with Homebrew, and it gets installed at the path `/opt/homebrew/bin/python3` to not conflict with the older system python3 at `/usr/bin/python3`. Likewise, people may have a virtualenv active in which case they want to use the venv's python, not this system python. So changing this to the following would let everyone run this as `./qlever.py` regardless of how they have python configured, I think:
```suggestion
#!/usr/bin/env python3
```